apps/domain/src/main/core/manager/user_manager.py

```diff
@@ -18,6 +18,17 @@ def __init__(self, database):
         self.roles = RoleManager(database)
         self.db = database
 
+    @property
+    def common_users(self) -> list:
```

Comment:
This fails if new roles are added and users are not part of the `User` role.

Mind that even the `User` role can be deleted.

The same goes for org users.

Maybe it is possible to retrieve this information by looking at the users permissions set?

"common users"
```json
{
 "can_triage_requests": false,
 "can_edit_settings": false,
 "can_create_users": false,
 "can_create_groups": false,
 "can_upload_data": false,
 "can_edit_roles": false,
 "can_manage_infrastructure": false
}
```

`org-level` users all users with different permissions.

Reply:
Fixed! :smiley: